data/battle_scripts_1.s

```diff
@@ -378,7 +378,7 @@ gBattleScriptsForMoveEffects::
 	.4byte BattleScript_EffectCoaching                @ EFFECT_COACHING
 	.4byte BattleScript_EffectHit                     @ EFFECT_LASH_OUT
 	.4byte BattleScript_EffectHit                     @ EFFECT_GRASSY_GLIDE
-	.4byte BattleScript_EffectRemoveTerrain           @ EFFECT_REMOVE_TERRAIN
+	.4byte BattleScript_EffectSteelRoller             @ EFFECT_STEEL_ROLLER
 	.4byte BattleScript_EffectHit                     @ EFFECT_DYNAMAX_DOUBLE_DMG
```

Comment:
Steel Roller's effect is unique in that it fails if there's no terrain to remove - not like Splintered Stormshards or Ice Spinner